lib/pages/editor-page.js

```diff
@@ -104,7 +104,7 @@ export default class EditorPage extends BaseContainer {
 
 	dismissMediaModal() {
 		const driver = this.driver;
-		const cancelSelector = webdriver.By.xpath( "//div[contains(@class, 'editor-media-modal')]//span[contains(@class, 'dialog__button-label') and contains(text(), 'Cancel' )]" );
+		const cancelSelector = webdriver.By.css( 'div.dialog__action-buttons > button:nth-child(1) > span' );
```

Comment:
Starting to take a look at this issue now. Is this referring to the firstSiteSelector above, or was there an issue with the cancelSelector, too? (It's showing the comment on the cancelSelector line, so I just wanted to be sure. Thanks!).